src/app/fdctl/Local.mk

```diff
@@ -54,6 +54,9 @@ fdctl: $(OBJDIR)/bin/fdctl
 $(OBJDIR)/bin/solana: solana/target/$(RUST_PROFILE)/solana
 	$(MKDIR) $(dir $@) && cp solana/target/$(RUST_PROFILE)/solana $@
 
+$(OBJDIR)/bin/solana-keygen: solana/target/$(RUST_PROFILE)/solana-keygen
```

Comment:
Prefer to use `fdctl keygen` maybe add a faucet keygen for it?

Reply:
I added the subcommand `fddev keygen pubkey <path-to-file>` which is used in the new test.